app/scripts/controllers/transactions.js

```diff
@@ -26,7 +26,7 @@ angular.module('openhimConsoleApp')
     $scope.settings.list = {};
     $scope.settings.list.tabview = 'same';
     $scope.settings.filter = {};
-    $scope.settings.filter.limit = 10;
+    $scope.settings.filter.limit = 100;
```

Comment:
No specific reason for the change. If im not mistaken @devcritter mentioned it might be better to show 100 records by default.  Should this rather default to the min amount available (10)? The user can always set their own default in their profile settings